docs/snippets/vue/button-component-with-proptypes.js.mdx

```diff
@@ -0,0 +1,31 @@
+```html
+<!-- Button.vue -->
```

Comment:
Is `button-component-with-proptypes.js.mdx` the right filename? Should it be called `button-component-with-proptypes.vue.mdx`? I don't quite remember how this all works.

Reply:
Yes it is, i was getting a compilation error locally, that's why i went with `js.mdx` don't know why that was happening. But one thing that did the trick was purge the cache and public folders and it was working. Fixed with latest commit